news/_posts/2021-05-24-prestashop-available-composer.md

```diff
@@ -11,6 +11,13 @@ tags:
 - development
 ---
 
+
+<div class="alert alert-note" role="alert">
+<h4><i class='icon-file'></i> Note</h4>
+We strongly recommend not to use in production.
```

Comment:
Well, why ? 😄

Reply:
It depends, if you use from dist option, yes you can, if you use from source, I agree that it shouldn't be used in production because of git files and upgrade process.

Reply:
Maybe we can provide this context somewhere :) (does not have to be here, maybe in doc and we add a link?)

Reply:
Yes, I agree, inside the doc is a better place :)